package.json

```diff
@@ -1,102 +1,107 @@
-      "github:html-next/vertical-collection#5c4fc6b18c75ec161eee39622dd0c068bff735f2",
-    "babel-eslint": "^8.0.1",
-    "babel-plugin-transform-async-to-generator": "^6.24.1",
-    "babel-plugin-transform-decorators-legacy": "^1.3.4",
-    "babel-plugin-transform-object-rest-spread": "^6.23.0",
-    "babel-preset-env": "^1.6.1",
-    "broccoli-asset-rev": "^2.4.5",
+    "@babel/plugin-proposal-decorators": "^7.8.3",
+    "@babel/plugin-proposal-object-rest-spread": "^7.8.3",
+    "@babel/plugin-transform-async-to-generator": "^7.8.3",
+    "@babel/preset-env": "^7.8.3",
+    "@ember/jquery": "^1.1.0",
+    "@ember/optional-features": "^1.1.0",
+    "@html-next/vertical-collection": "github:html-next/vertical-collection#4c787b2cd92ec6fd548b52c4e22d8e0e0d0b85d3",
+    "babel-eslint": "^10.0.3",
+    "broccoli-asset-rev": "^3.0.0",
     "downloadjs": "^1.4.7",
-    "ember-adcssy": "^1.2.1",
-    "ember-ajax": "^3.0.0",
-    "ember-basic-dropdown": "^1.0.2",
-    "ember-can": "^1.1.1",
-    "ember-changeset": "^1.5.0",
-    "ember-changeset-validations": "^1.3.1",
-    "ember-cli": "~2.18.0",
+    "ember-ajax": "^5.0.0",
+    "ember-auto-import": "^1.5.3",
+    "ember-basic-dropdown": "^2.0.11",
+    "ember-can": "^2.1.0",
+    "ember-changeset": "^2.2.4",
+    "ember-changeset-validations": "^2.2.1",
+    "ember-cli": "~3.12.0",
     "ember-cli-app-version": "^3.2.0",
-    "ember-cli-babel": "^6.6.0",
-    "ember-cli-chai": "^0.5.0",
-    "ember-cli-chart": "^3.4.0",
-    "ember-cli-code-coverage": "^0.4.2",
-    "ember-cli-dependency-checker": "^3.0.0",
-    "ember-cli-eslint": "^5.0.0",
-    "ember-cli-htmlbars": "^3.0.0",
-    "ember-cli-htmlbars-inline-precompile": "^0.4.4",
-    "ember-cli-inject-live-reload": "^1.4.1",
-    "ember-cli-mirage": "^0.3.4",
-    "ember-cli-mocha": "^0.15.0",
+    "ember-cli-babel": "^7.13.2",
+    "ember-cli-chart": "^3.5.0",
+    "ember-cli-code-coverage": "^1.0.0-beta.8",
+    "ember-cli-dependency-checker": "^3.2.0",
+    "ember-cli-eslint": "^5.1.0",
+    "ember-cli-htmlbars": "^4.2.0",
+    "ember-cli-inject-live-reload": "^1.8.2",
+    "ember-cli-mirage": "^1.1.6",
     "ember-cli-moment-shim": "^3.5.0",
-    "ember-cli-nouislider": "^1.0.0",
-    "ember-cli-postcss": "^4.0.0",
+    "ember-cli-nouislider": "^1.1.0",
     "ember-cli-release": "^1.0.0-beta.2",
-    "ember-cli-sass": "^7.0.0",
-    "ember-cli-shims": "^1.1.0",
-    "ember-cli-sri": "^2.1.0",
+    "ember-cli-sass": "^10.0.0",
+    "ember-cli-shims": "^1.2.0",
+    "ember-cli-sri": "^2.1.1",
     "ember-cli-test-loader": "^2.1.0",
     "ember-cli-uglify-es": "^1.0.4",
-    "ember-component-css": "^0.6.4",
-    "ember-composable-helpers": "^2.0.3",
-    "ember-concurrency": "^0.8.12",
-    "ember-data": "~2.18.0",
-    "ember-export-application-global": "^2.0.0",
-    "ember-fetch": "^5.1.1",
-    "ember-font-awesome": "^3.0.5",
-    "ember-load-initializers": "^1.0.0",
-    "ember-math-helpers": "^2.2.3",
+    "ember-cli-update": "^0.47.1",
```

Comment:
Not necessary as dependency

Reply:
Hmmm... This one I like as it extends the `ember` command.

I'll get back to this one later.